OpenOversight/app/main/forms.py

```
@@ -514,6 +514,27 @@ class OOIdForm(Form):
 
 
 class IncidentForm(DateFieldForm):
+    # In line validator for the officers field. Check to see if the officers are within the selected department
+    def validate_officers(self, field: FieldList) -> None:
+        department = self.department.data
+
+        if not department:
+            return
+
+        department_id = department.id
+        department_name = department.name
+
+        for field_data in field.data:
+            officer = Officer.query.get(field_data["oo_id"])
+
+            if not officer:
+                return None
```

I think we should raise an error here instead of failing silently